frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -500,6 +500,54 @@ export const getEducationType = (
     `Invalid educationType ${educationTypeId}`,
   );
 
+export const OperationalRequirementSearchDescription = defineMessages({
+  [OperationalRequirement.ShiftWork]: {
+    defaultMessage: "Shift-work",
+    description: "The operational requirement described as shift work.",
+  },
+  [OperationalRequirement.OnCall]: {
+    defaultMessage: "24/7 on call-shifts",
+    description: "The operational requirement described as 24/7 on-call.",
+  },
+  [OperationalRequirement.Travel]: {
+    defaultMessage: "Travel as required",
+    description: "The operational requirement described as travel as required.",
+  },
+  [OperationalRequirement.TransportEquipment]: {
+    defaultMessage:
+      "Transport, lift, and set down equipment weighing up to 20kg",
+    description:
+      "The operational requirement described as transport equipment up to 20kg.",
+  },
+  [OperationalRequirement.DriversLicense]: {
+    defaultMessage:
+      "Valid driver's license or personal mobility to the degree normally associated with possession of a valid driver's license",
+    description: "The operational requirement described as driver's license.",
+  },
+  [OperationalRequirement.WorkWeekends]: {
```

Comment:
Sorry, I'm a bit confused here.  The descriptions are quite different from the wireframe.  There's not even the same number!  @Jerryescandon could you help us out, please?  Also, does the order of the options matter?
![image](https://user-images.githubusercontent.com/8978655/174893086-5ecc6584-9975-4f15-985a-ce46ce723ee8.png)

Reply:
as for order, I matched it to the form on the /profile

Reply:
Would it be possible to update the design?

Reply:
Actually the ones I have on profile (Figma) are these. and they already match. 
![image](https://user-images.githubusercontent.com/16977254/174896320-5ae19ca5-5a84-41e1-b489-b2a16a2d7127.png)

Reply:
Yall wanna hop on a call for this?

Reply:
okey